corinaorasan171015/AppNavigation
# Add sharing with an intent

## Description
Add intents to allow users to share through their hashtag Android Trivia awesomeness

### [Task]
https://classroom.udacity.com/courses/ud9012/lessons/7466f670-3d47-4b60-8f6a-0914ce58f9ad/concepts/e8ecc9a9-b393-4df2-bb5c-72827cc083a4

## Steps to Test or Reproduce
1.Start the app
2.Press Play button
3.Check the answer correctly 
4.Press the share button
5.Choose where you want to share
